<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateEntryDialog</class>
 <widget class="QDialog" name="CreateEntryDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Sync Entry</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="pathLabel">
     <property name="text">
      <string>&amp;Path:</string>
     </property>
     <property name="buddy">
      <cstring>pathLineEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLineEdit" name="pathLineEdit">
       <property name="placeholderText">
        <string>Enter a path or path pattern</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="selectButton">
       <property name="popupMode">
        <enum>QToolButton::MenuButtonPopup</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="modeLabel">
     <property name="text">
      <string>&amp;Mode:</string>
     </property>
     <property name="buddy">
      <cstring>modeComboBox</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="modeComboBox"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="hiddenFilesLabel">
     <property name="text">
      <string>&amp;Hidden Files:</string>
     </property>
     <property name="buddy">
      <cstring>hiddenFilesCheckBox</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QCheckBox" name="hiddenFilesCheckBox">
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="caseSensitiveLabel">
     <property name="text">
      <string>&amp;Case Sensitive:</string>
     </property>
     <property name="buddy">
      <cstring>caseSensitiveCheckBox</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QCheckBox" name="caseSensitiveCheckBox">
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>&amp;Extras:</string>
     </property>
     <property name="buddy">
      <cstring>listWidget</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QListWidget" name="listWidget">
     <property name="contextMenuPolicy">
      <enum>Qt::ActionsContextMenu</enum>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QToolButton" name="addButton">
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonFollowStyle</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="removeButton">
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonFollowStyle</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="action_Add_Extra">
   <property name="icon">
    <iconset theme="list-add">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Add Extra</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+Ins</string>
   </property>
  </action>
  <action name="action_Remove_Extra">
   <property name="icon">
    <iconset theme="list-remove">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Remove Extra</string>
   </property>
   <property name="shortcut">
    <string notr="true">Del</string>
   </property>
  </action>
  <action name="actionSelect_File">
   <property name="icon">
    <iconset theme="dialog-object-properties">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Select &amp;File</string>
   </property>
  </action>
  <action name="actionSelect_Directory">
   <property name="icon">
    <iconset theme="dialog-object-properties">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Select &amp;Directory</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CreateEntryDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CreateEntryDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
